source/appendices/e-audit-requirements-and-tests/05-book-usage-tests.rst

```diff
@@ -0,0 +1,98 @@
+
+These tests apply to the Platform Report, Database Report, Title Report and Item Report. The PR_P1, DR_D1, TR_B1 and TR_B3 Standard Views of the COUNTER Reports will be deemed COUNTER-compliant if the metrics and Data_Types match those in the relevant COUNTER Report, with the appropriate aggregation. 
+
+Unique_Title_Investigations and Unique_Title_Requests are referenced in these audit tests. These MUST NOT appear in the Item Report.
+
+These audit tests specify minimum numbers of Books and Book_Segments to be tested. Where a platform has fewer than the required number of Books, the auditor MUST make at least one request for each Book on the platform, and where necessary undertake duplicate actions to reach the required threshold.
+
+Audit test requirements vary depending on the set up of the platform, as indicated by the Options within the tests below.
+
+
+E.5.1 Books Available as Book_Segments
+""""""""""""""""""""""""""""""""""""""
+
+These tests apply where users can access books segment-by-segment.
+
+**Option 1**: Report provider offers Book_Segments under only one Access_Type.
+
+The auditor MUST request 70 Book_Segments across 7 different Books.
+
+This MUST result in 70 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 7 each of Unique_Title_Investigations and Unique_Title_Requests with the appropriate Access_Type.
+
+**Option 2**: Report provider offers Book_Segments under two different Access_Types.
+
+The auditor MUST request 50 Book_Segments across 5 different Books, from each Access_Type represented on the platform.
+
+This MUST result in 50 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 5 each of Unique_Title_Investigations and Unique_Title_Requests with each Access_Type.
+
+The Access_Type combinations might be: Controlled plus Open, Controlled plus Free_To_Read, or Open plus Free_To_Read.
+
+**Option 3**: Report provider offers Book_Segments under all three Access_Types (Controlled, Open and Free_To_Read).
+
+The auditor MUST request
+
+* 40 Book_Segments across 4 different Books with Access_Type Controlled.
+* 40 Book_Segments across 4 different Books with Access_Type Open.
+* 20 Book_Segments across 2 different Books with Access_Type Free_To_Read.
+
+This MUST result in 40 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 4 each of Unique_Title_Investigations and Unique_Title_Requests with Access_Type Controlled; the same again for Access_Type Open; and 20 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 2 each of Unique_Title_Investigations and Unique_Title_Requests with Access_Type Free_To_Read.
+
+
+E.5.2 Books Available as Both Book_Segments and Whole Books
```

Comment:
This and the next section title have to be adapted - E.5.2 is for whole books (the book segment case for report providers that offer both whole books and segments is covered by E.5.1), when segments can be identified, and E.5.3 for whole books only, when segments cannot be identified.